pkgs/top-level/python-packages.nix

```diff
@@ -3706,8 +3706,6 @@ in {
 
   git-sweep = callPackage ../development/python-modules/git-sweep { };
 
-  glances = callPackage ../development/python-modules/glances { };
```

Comment:
Since the move will cause errors on existing setups I think it would be nice to add a `throw` statement here and explain that the package was moved to the top-level and is now available as `glances`.

Reply:
agreed